src/tsbootstrap/__init__.py

```diff
@@ -1,3 +1,6 @@
+from importlib.metadata import version
+__version__ = version("tsbootstrap")
```

Comment:
hm, why this way and not explicitly? Does this reduce the number of places to update?

Reply:
Yes I wanted to use only the information from the pyproject.toml file. However, if you prefer an explicit solution, I am happy to change it

Reply:
I think this is better, if it works, as it reduces the number of update (and hence failure) points

Reply:
should we do this in `sktime` too?